### PR TITLE
fix inputs and harvest

### PR DESCRIPTION
**Pull request by mendesfabio**
FIxes

- https://github.com/Giveth/GIVeconomy/issues/348
- https://github.com/Giveth/GIVeconomy/issues/360

### CODE REVIEW

**Review by MohammadPCh — CHANGES_REQUESTED** [review]

File: src/components/modals/HarvestAll.sc.tsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ export const NothingToHarvest = styled(H6)`
 
 export const Pending = styled(Row)`
 	margin: 0 auto 16px;
-	width: 316px;
+	width: 100%;
```
[the remark this answers is not in the document]
Comment (mendesfabio) [reply src/components/modals/HarvestAll.sc.tsx:161]: Hardcoding Pending's width will break other components such as Stake/Unstake NFTs modal. It should be 100% and #67 fixes the Harvest button width.

[the remark this answers is not in the document]
Comment (MohammadPCh) [reply src/components/modals/HarvestAll.sc.tsx:161]: so we should fix the Pending component everywhere we use it.

[the remark this answers is not in the document]
Comment (mendesfabio) [reply src/components/modals/HarvestAll.sc.tsx:161]: we've done it

[the remark this answers is not in the document]
Comment (MohammadPCh) [reply src/components/modals/HarvestAll.sc.tsx:161]: Ok, Thanks.

**Review by MohammadPCh — APPROVED** [review]

**Pull request merged**